src/Compilers/CSharp/Portable/Symbols/AliasSymbol.cs

```diff
@@ -377,20 +377,23 @@ private NamespaceSymbol ResolveExternAliasTarget(BindingDiagnosticBag diagnostic
             BindingDiagnosticBag diagnostics,
             ConsList<TypeSymbol>? basesBeingResolved)
         {
+            var reportedError = false;
             if (usingDirective.UnsafeKeyword != default)
             {
-                MessageID.IDS_FeatureUsingTypeAlias.CheckFeatureAvailability(diagnostics, usingDirective, usingDirective.UnsafeKeyword.GetLocation());
+                reportedError = !MessageID.IDS_FeatureUsingTypeAlias.CheckFeatureAvailability(diagnostics, usingDirective, usingDirective.UnsafeKeyword.GetLocation());
             }
             else if (usingDirective.NamespaceOrType is not NameSyntax)
             {
-                MessageID.IDS_FeatureUsingTypeAlias.CheckFeatureAvailability(diagnostics, usingDirective.NamespaceOrType);
+                reportedError = !MessageID.IDS_FeatureUsingTypeAlias.CheckFeatureAvailability(diagnostics, usingDirective.NamespaceOrType);
             }
 
             var syntax = usingDirective.NamespaceOrType;
             var flags = BinderFlags.SuppressConstraintChecks | BinderFlags.SuppressObsoleteChecks;
             if (usingDirective.UnsafeKeyword != default)
             {
-                this.CheckUnsafeModifier(DeclarationModifiers.Unsafe, usingDirective.UnsafeKeyword.GetLocation(), diagnostics);
+                if (!reportedError)
```

Comment:
I don't think the `CheckUnsafeModifier` should be made conditional.
If we make it conditional, then a new error appears after I upgrade my language version. That seems unnecessary.

Reply:
ok.  i can change this to give more errors.

Reply:
It looks like this wasn't addressed

Reply:
Misread the diff. Thanks!